react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/BasePaywallViewManager.kt

```diff
@@ -9,9 +9,18 @@ import com.facebook.react.uimanager.ThemedReactContext
 import com.facebook.react.uimanager.annotations.ReactProp
 import com.facebook.react.uimanager.events.RCTEventEmitter
 import com.revenuecat.purchases.hybridcommon.ui.PaywallListenerWrapper
+import com.revenuecat.purchases.ui.revenuecatui.ExperimentalPreviewRevenueCatUIPurchasesAPI
+import com.revenuecat.purchases.ui.revenuecatui.fonts.CustomFontProvider
 
 internal abstract class BasePaywallViewManager<T : View> : SimpleViewManager<T>() {
 
+    companion object PropNames {
+        private const val PROP_OPTIONS = "options"
+        private const val OFFERING = "offering"
```

Comment:
Maybe this could be `OPTION_OFFERING`, and `OFFERING_IDENTIFIER`, and `OPTION_FONT_FAMILY`